packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2836,17 +2936,26 @@ void main() {
     Logger: () => logger,
   });
 
-  testUsingContext('newly created plugin has min flutter sdk version as 3.3.0', () async {
+  testUsingContext('newly created package / plugin / module has min flutter sdk version 3.10.0', () async {
     Cache.flutterRoot = '../..';
 
     final CreateCommand command = CreateCommand();
     final CommandRunner<void> runner = createTestCommandRunner(command);
-    await runner.run(<String>['create', '--no-pub', '--template=plugin', projectDir.path]);
-    final String rawPubspec = await projectDir.childFile('pubspec.yaml').readAsString();
-    final Pubspec pubspec = Pubspec.parse(rawPubspec);
-    final Map<String, VersionConstraint?> env = pubspec.environment!;
-    expect(env['flutter']!.allows(Version(3, 3, 0)), true);
-    expect(env['flutter']!.allows(Version(3, 2, 9)), false);
+    final List<String> templates = <String>['plugin', 'plugin_ffi', 'package', 'module'];
```

Comment:
I can't pass `package_ffi` to the template option. Any ideas on how to test that one?